modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -461,7 +462,7 @@ Type::
 | `disableAlerts` is a list of alerts that should be disabled.
 Possible values are: +
 
-`NetObservDroppedFlows` is triggered when the eBPF agent is missing packets or flows, such as when the eBPF hashmap is busy or full, or the capacity limiter is triggered. +
+`NetObservDroppedFlows`, which is triggered when the eBPF agent is missing packets or flows, such as when the BPF hashmap is busy or full, or the capacity limiter is being triggered. +
```

Comment:
Possible to split the example cases off as a second sentence?

Reply:
Thanks but I think its OK as it is.